src/components/ProductFilterSelect/ProductFilterSelect.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import "./styles.css";
+
+const ProductFilterSelect = ({ onChange, value }) => {
+  const [filters] = useState(["Mais vendido", "Menor preço", "Maior preço"]);
+
+  const handleFilter = () => {
+    // Lógica para aplicar o filtro
+  };
+
+  return (
+    <div className="SCCustomSelect">
+      <select name="" id="SCFilterProduct" value={value} onChange={onChange}>
+        <option value={value || ""}>Selecione um filtro</option>
+        {filters.map((value, index) => (
+          <option key={index} value={value}>
+            {value}
+          </option>
+        ))}
+      </select>
+      <button onClick={handleFilter}>Filtrar</button>
```

Comment:
Como vc está usando a propriedade "pointer-events: none"  no CSS, que desabilita as funções do mouse no botão, acredito que essa propriedade onClick não seria necessária

Reply:
Na verdade imagino que é melhor substituir o botão por um label, a princípio coloquei o botão pensando em uma semântica, mas ao selecionar um filtro a ideia é que já realize a filtragem sem apertar nesse botão. Então ele está somente visual, acho que não faz muito sentido manter um botão que não realiza nenhuma ação. O que acha dessa substituição por um label?

Reply:
se não for usar pointer-events não usa button ... lembra da acessibilidade. 
Se a filtragem já irá ocorrer no onChange do select ... o botão nem precisa existir, nem mesmo como label. Daí é um ajuste que em um projeto junto com os designers já seria discutido para não ter o botão.

Reply:
> 

Acho interessante a label sim, a questão que a Leca pontuou também é outro que eu não me lembrava em relação a acessibilidade.